src/presentational-components/shared/toolbar.js

```diff
@@ -7,9 +7,9 @@ import { pickBy } from 'lodash';
 import { selectedRows, calculateChecked, debouncedFetch, firstUpperCase } from '../../helpers/shared/helpers';
 import { calculateOffset, calculatePage, defaultSettings } from '../../helpers/shared/pagination';
 
-export const paginationBuilder = (pagination = {}, fetchData = () => undefined, filterValue = '', sortBy = '') => ({
+export const paginationBuilder = (pagination = {}, fetchData, filterValue = '', sortBy = '') => ({
   ...pagination,
-  itemCount: pagination.count,
+  count: pagination.count,
```

Comment:
this needs to be reverted. Pagination PF component needs the property to be named like this